plugins/sql-language/src/main/java/org/polypheny/db/sql/language/validate/DelegatingScope.java

```diff
@@ -231,268 +229,259 @@ public SqlQualified fullyQualify( SqlIdentifier identifier ) {
         }
 
         final SqlIdentifier previous = identifier;
-        final NameMatcher nameMatcher = validator.catalogReader.nameMatcher();
+        final NameMatcher nameMatcher = Snapshot.nameMatcher;
         String columnName;
         final String tableName;
         final SqlValidatorNamespace namespace;
-        switch ( identifier.names.size() ) {
-            case 1: {
-                columnName = identifier.names.get( 0 );
-                final Map<String, ScopeChild> map = findQualifyingTableNames( columnName, identifier, nameMatcher );
-                switch ( map.size() ) {
-                    case 0:
-                        if ( nameMatcher.isCaseSensitive() ) {
-                            final NameMatcher liberalMatcher = NameMatchers.liberal();
-                            final Map<String, ScopeChild> map2 = findQualifyingTableNames( columnName, identifier, liberalMatcher );
-                            if ( !map2.isEmpty() ) {
-                                final List<String> list = new ArrayList<>();
-                                for ( ScopeChild entry : map2.values() ) {
-                                    final AlgDataTypeField field = liberalMatcher.field( entry.namespace.getRowType(), columnName );
-                                    list.add( field.getName() );
-                                }
-                                Collections.sort( list );
-                                throw validator.newValidationError( identifier, Static.RESOURCE.columnNotFoundDidYouMean( columnName, Util.sepList( list, "', '" ) ) );
+        if ( identifier.names.size() == 1 ) {
+            columnName = identifier.names.get( 0 );
+            final Map<String, ScopeChild> map = findQualifyingTableNames( columnName, identifier, nameMatcher );
+            switch ( map.size() ) {
+                case 0:
+                    if ( nameMatcher.isCaseSensitive() ) {
```

Comment:
Shouldn't this be the job of the snapshot?

Reply:
Removed as part of: https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1518969914